proto/sequencerapis/astria/sequencer/v1alpha1/block.proto

```diff
@@ -116,3 +117,16 @@ message FilteredSequencerBlock {
   // the rollup transactions.
   astria.sequencer.v1alpha1.Proof rollup_ids_proof = 6;
 }
+
+// A piece of data that is sent to a rollup execution node.
+//
+// The data can be either sequenced data (originating from a `SequenceAction`
+// submitted by a user) or a `Deposit` originating from a `BridgeLockAction`.
+//
+// The rollup node receives this type from conductor and must decode them accordingly.
+message RollupData {
+  oneof value {
+    bytes sequenced_data = 1;
```

Comment:
Are these just opaque bytes?

Reply:
yes, they're what's in sequence actions

Reply:
What if we called this `opaque_transaction_bytes` to really bring home what this is supposed to be?